typescript/web/src/components/export-button/export-modal/modal-header.tsx

```diff
@@ -24,27 +23,14 @@ export const ModalHeader = () => {
           Your dataset contains {imagesNumber} images and {labelsNumber} labels.
         </Text>
       </Skeleton>
-      <Box display="inline-block" width="100%" pl={8} pr={8} pt={4}>
-        {
-          // eslint-disable-next-line eqeqeq
-          numberUndefinedLabelsOfDataset != null &&
-            numberUndefinedLabelsOfDataset !== 0 && (
-              <Alert status="warning" borderRadius={5}>
-                <AlertIcon />
-                <AlertTitle mr={2} fontSize={15} whiteSpace="nowrap">
-                  Missing Class
-                </AlertTitle>
-                <AlertDescription fontSize={13} fontWeight="medium">
-                  {`${numberUndefinedLabelsOfDataset} ${
-                    numberUndefinedLabelsOfDataset === 1 ? "label" : "labels"
-                  } ${
-                    numberUndefinedLabelsOfDataset === 1 ? "has" : "have"
-                  } no class assigned. Only labels with a class
-            are exported.`}
-                </AlertDescription>
-              </Alert>
-            )
-        }
+      <Box width="100%" pl={8} pr={8} pt={4}>
+        <Alert status="info" borderRadius={5}>
+          <AlertIcon />
+          <AlertTitle fontSize={15}>About your export</AlertTitle>
+          <AlertDescription fontSize={13} fontWeight="medium">
+            Labels without a class are not exported
+          </AlertDescription>
+        </Alert>
```

Comment:
I'll be displayed for every user. Even when they don't have any label with a missing label-class.
Do we really need to show this message with such an emphasis?

Reply:
Maybe, maybe not. We can also completely remove it.

Reply:
Let's remove it completely 👍

Reply:
I have removed it